app/models/user.rb

```diff
@@ -9,13 +9,13 @@ class User < ActiveRecord::Base
   include Taxonomix
   include DirtyAssociations
   include UserTime
-  audited :except => [:last_login_on, :password, :password_hash, :password_salt, :password_confirmation]
+  audited :except => [:last_login_on, :password, :password_salt, :password_confirmation]
```

Comment:
I think that's a separate change, whether or not that's written in the issue description.